src/main/java/stelle/exception/StelleException.java

```diff
@@ -0,0 +1,12 @@
+package stelle.exception;
+
+/**
+ * Parent for all exceptions related to the chatbot.
+ * @author Lee Ze Hao (A0276123J)
+ */
+
+public class StelleException extends RuntimeException {
```

Comment:
Good job on adding so many different exceptions to cover different issue the program could face!!